lib/mapper/product_mapper.rb

```diff
@@ -19,7 +21,7 @@ def self.map_payload(product, additional_data = {}, track_inventory = true, defa
                   meta_keywords: self.meta_keywords(product),
                   name: name,
                   page_title: product['page_title'] || '',
-                  preorder_message: self.get_availability(product) == 'preorder' ? product['availability'] : '',
+                  preorder_message: self.get_availability(product) == 'preorder' ? product['offers'][0]['availability'] : '',
```

Comment:
@Jhalter5Stones I don't think this is the right source for this field. How is this being set by the Acumen agent?